lib/routes/admin-api/import.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Controller = require('../controller');
+const { ADMIN } = require('../../permissions');
+const extractUser = require('../../extract-user');
+const { handleErrors } = require('./util');
+
+class ImportController extends Controller {
+    constructor(config) {
+        super(config);
+        this.post('/', this.import, ADMIN);
+    }
+
+    async import(req, res) {
+        const userName = extractUser(req);
+        const { drop } = req.query;
+        const data = req.body;
```

Comment:
Should accept data sent as a file upload.

Reply:
I agree

Reply:
But this would require multipart/form-data. Right now we're only using `bodyparser.json()` middleware.

Reply:
Yeaæs indeed. But I don't see how we can provide a useful user experience if the user can't upload a file? Should they copy/paste a long json content block? We can add this layter though when we add the UI feature